src/screens/About/Team/TeamSection.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment } from 'react';
+import { teams, teamsWithTitles } from 'cms-content/team';
+import { AboutHeading3 } from '../About.style';
+import { sortBy } from 'lodash';
+import Grid from '@material-ui/core/Grid';
+import ActiveMember from './ActiveMember';
+import AlumniMember from './AlumniMember';
+
+const TeamSection = () => {
+  return (
+    <Fragment>
+      {teams.map(team => {
+        const alphabetizedTeam = sortBy(
+          team.teamMembers,
+          member => member.fullName,
+        );
+        const teamToMap =
```

Comment:
nit: I think `teamMembers` might be more clear here - took me a sec to realize that what "to map" was for